pywt/_multidim.py

```diff
@@ -58,8 +58,12 @@ def dwt2(data, wavelet, mode='symmetric', axes=(-2, -1)):
 
     """
     axes = tuple(axes)
+    data = np.asarray(data)
```

Comment:
Makes it consistent with other functions in this file, so +1. Would be good to change the parameter documentation from ``ndarray`` to ``array_like`` though, for ``dwt2``/``dwtn``/``swt2``